nilearn/plotting/displays/_slicers.py

```diff
@@ -353,16 +368,40 @@ def add_contours(self, img, threshold=1e-6, filled=False, **kwargs):
             "colors", which is one color or a list of colors for
             these contours.
 
+        cbar_vmin : :obj:`float`, optional
+            Minimal value for the colorbar. If None, the minimal value
+            is computed based on the data.
+
+        cbar_vmax : :obj:`float`, optional
+            Maximal value for the colorbar. If None, the maximal value
+            is computed based on the data.
+
         Notes
         -----
         If colors are not specified, default coloring choices
         (from matplotlib) for contours and contour_fillings can be
         different.
 
+        There is not point at using the parameters colorbar and levels at the same time.
+        Then, if one does so, it will raise an error.
+        It is possible to use add_contours with parameter levels and without colorbar,
+        and then use it again with parameter colorbar and without levels,
+        but it is not relevant and it will display something that doesn't make sense.
```

Comment:
```suggestion
        There is not point at using the parameters colorbar
        and levels at the same time.
        Then, if one does so, it will raise an error.
        It is possible to use add_contours
        with parameter levels and without colorbar,
        and then use it again
        with parameter colorbar and without levels,
        but it is not relevant
        and it will display something that doesn't make sense.
```

Reply:
should pacify the style check